### PR TITLE
Fixes #195 add chronological sorting to home page, user/org profile page

### LINKED ISSUE #195: Allow to sort petitions chronologically or reverse order in home page or profile pages
Today it's not even consistent : 
* home page shows petitions in reverse chronological order
* "all petitions" page shows them in chronological order.

### PR DESCRIPTION
**Pull request by toshihidetagami**
Fixes #195

- This pull request adds chronological sorting to the Home page, User profile page, organization page.
- "creation_date" is used for sorting
- I initially planned to use drop-down for sorting. But I chose "segmented button" because I think it looks nicer on UI and the code will be simpler.

I am happy to fix or refactor if there are any problems with code, UI, etc.

**Orgnization Profile Page**
Screen Shot 2020-10-17 at 8 53 47 PM

**User Profile Page**
Screen Shot 2020-10-17 at 8 54 09 PM

**Home Page**
Screen Shot 2020-10-17 at 8 54 45 PM

### CODE REVIEW

File: pytition/petition/templates/petition/index.html [hunk 1]
```diff
@@ -18,10 +18,18 @@ <h1 class="jumbotron-heading">{% trans "Welcome to Pytition"%}</h1>
...
 </section>
-<section class="petition-list bg-light">
+<section id="petition-list-id" class="petition-list bg-light">
   <div class="text-center container">
     <h2>{% trans "View latest petitions" %}</h2>
-    {% include 'petition/generic_petition_list.html' with petitions=petitions %}
+    <div class="d-flex justify-content-end">
+      <div class="btn-group" role="group" aria-label="sort-buttons">
+          <a class="btn {% if sort == 'desc' %} btn-secondary {% else %} btn-outline-secondary {% endif %}"
+              href="{% url "index" %}?sort=desc#petition-list-id" role="button">Last Created</a>
```
Comment (fallen) [inline pytition/petition/templates/petition/index.html:27]: You need to use {% trans "Last Created" %} for all texts so that they can be translated

**Review by fallen — CHANGES_REQUESTED** [review]
This PR looks very good, thanks!
Pictures are :+1: 
One minor change to allow texts to be translated and I can merge this :)

**Comment by toshihidetagami** [comment]
I just added a commit for the translation!

**Review by fallen — APPROVED** [review]
Looks good to me, merging!
Thanks :+1:

**Pull request merged**